config/config.go

```diff
@@ -85,7 +85,8 @@ type PoliciesConfig struct {
 	// If you set this value to `true`, then the id parameter in a stored policy (or imported policy using the Dashboard API), will be used instead of the internal ID.
 	//
 	// This option should only be used when moving an installation to a new database.
-	AllowExplicitPolicyID bool `json:"allow_explicit_policy_id"`
+	AllowExplicitPolicyID bool   `json:"allow_explicit_policy_id"`
+	PolicyPath            string `json:"policy_path"`
```

Comment:
Do not forget adding a comment to all fields, since it will be used to auto-generate docs.

Reply:
Oh didn't know that, adding it